app/import_functions_test.go

```diff
@@ -3128,6 +3127,7 @@ func TestImportImportPost(t *testing.T) {
 		posts, nErr := th.App.Srv().Store.Post().GetPostsCreatedAt(channel.Id, now)
 		require.NoError(t, nErr)
 		require.Len(t, posts, 2, "Unexpected number of posts found.")
+		th.TestLogger.Flush()
```

Comment:
I couldn't reproduce the failure on my local setup. 

From the traceback in the jira ticket (https://mattermost.atlassian.net/browse/MM-44922) , it looks like the log line is present, but the check happens before the log line is printed. My guess is that this is happening because of buffering, so flushing the logs before the assertion.

Reply:
Yes, this is exactly what's happening: https://github.com/mattermost/mattermost-server/pull/20484